recaptchaenterprise/samples/snippets/create_mfa_assessment.py

```diff
@@ -0,0 +1,101 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START recaptcha_enterprise_mfa_assessment]
+
+from google.cloud import recaptchaenterprise_v1
+from google.cloud.recaptchaenterprise_v1 import Assessment
+
+
+def create_mfa_assessment(
+    project_id: str, recaptcha_site_key: str, token: str, recaptcha_action: str, hashed_account_id: str,
+        email: str, phone_number: str
+) -> None:
+    """  Creates an assessment to obtain Multi-Factor Authentication result.
+         If the result is unspecified, sends the request token to the caller to initiate MFA challenge.
+    Args:
+        project_id: GCloud Project ID
+        recaptcha_site_key: Site key obtained by registering a domain/app to use recaptcha services.
+        token: The token obtained from the client on passing the recaptchaSiteKey.
+               To get the token, integrate the recaptchaSiteKey with frontend. See,
+               https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#frontend_integration_score
+        recaptcha_action: The action name corresponding to the token.
+        hashed_account_id: Create hashedAccountId from user identifier.
+                           It's a one-way hash of the user identifier: HMAC SHA 256 + salt
+        email: Email id of the user to trigger the MFA challenge.
+        phone_number: Phone number of the user to trigger the MFA challenge.
+    """
+
+    client = recaptchaenterprise_v1.RecaptchaEnterpriseServiceClient()
+
+    # Set the properties of the event to be tracked.
+    event = recaptchaenterprise_v1.Event()
+    event.site_key = recaptcha_site_key
+    event.token = token
+    event.hashed_account_id = hashed_account_id
+
+    # Set the email address and the phone number to trigger/ verify the MFA challenge.
+    endpoint_verification_info = recaptchaenterprise_v1.EndpointVerificationInfo()
+    endpoint_verification_info.email_address = email
+    endpoint_verification_info.phone_number = phone_number
+
+    account_verification_info = recaptchaenterprise_v1.AccountVerificationInfo()
+    account_verification_info.endpoints = endpoint_verification_info
+
+    assessment = recaptchaenterprise_v1.Assessment()
+    assessment.event = event
+    assessment.account_verification = account_verification_info
+
+    project_name = f"projects/{project_id}"
+
+    # Build the assessment request.
+    request = recaptchaenterprise_v1.CreateAssessmentRequest()
+    request.assessment = assessment
+    request.parent = project_name
+
+    # Check integrity of the response.
+    response = client.create_assessment(request)
+    if not verify_response_integrity(response, recaptcha_action):
+        raise Exception("Failed to verify token integrity.")
```

Comment:
Can we change it to a more specific error? Even `RuntimeError` would be better, so that if someone wants to catch this error, it's easier than catching everything.

Reply:
Sure, modified it to Runtime error. Thanks